M2/Macaulay2/m2/examples.m2

```diff
@@ -170,10 +171,24 @@ extractExamples = docBody -> (
 -----------------------------------------------------------------------------
 
 examples = method(Dispatch => Thing)
-examples Hypertext := dom -> raise(stack extractExamplesLoop dom, -1)
+examples Hypertext := h -> (
+    ex := extractExamplesLoop h;
+    if #ex == 0 then "-- no examples" else stack ex)
```

Comment:
Does this not get used in generating documentation? Does it not have unintended consequences?

Reply:
Ooh, you're right -- it does!  We only call `examples` once in Core -- in `getExampleOutput` when we can't find the results anywhere else and we need to generate them on demand.  Adding these comments messes that all up -- good catch!  I think this is likely the source of the `ofClass` premature termination errors I'm getting.

Reply:
I've modified things so that the exported `examples` that users will actually call has the extra output with tag and location information, but there's now an unexported `examples'` that `getExamplesOutput` calls with the original behavior.